src/base64Utils.js

```diff
@@ -4,33 +4,38 @@
  * SPDX-License-Identifier: MIT
  */
 
+#if SINGLE_FILE == 2 && !WASM_ASYNC_COMPILATION
+#error SINGLE_FILE == 2 does not work with synchronous WASM compilation (because DecompressionStream is async by nature)
```

Comment:
Hopefully we can remove this error if we simply the use of `DecompressionStream` the default whenever async is enabled.

If you do  want to keep this error we normally report incompatible settings in `link.py`